ACG/AutoRouter.py

```diff
@@ -481,8 +490,24 @@ def cardinal_router(self,
         manh_point_list = self.manhattanize_point_list(initial_direction=current_dir,
                                                        initial_point=current_point,
                                                        points=points)
+        # print (manh_point_list)
+        del_idx = []
+        for i in range(len(manh_point_list) - 2):
```

Comment:
I think this is replicating the `process_manh()` function you wrote below? Can we avoid repeating this section of code?